modules/command/common/handler_ids_cleaner_command.js

```diff
@@ -0,0 +1,75 @@
+const path = require('path');
+const fs = require('fs');
+const Command = require('../command');
+const Models = require('../../../models/index');
+const constants = require('../../constants');
+
+/**
+ * Increases approval for Bidding contract on blockchain
+ */
+class HandlerIdsCleanerCommand extends Command {
+    constructor(ctx) {
+        super(ctx);
+        this.logger = ctx.logger;
+        this.fileService = ctx.fileService;
+    }
+
+    /**
+     * Executes command and produces one or more events
+     * @param command
+     */
+    async execute(command) {
+        const timeToBeDeleted = Date.now() - constants.HANDLER_IDS_COMMAND_CLEANUP_TIME_MILLS;
+        await Models.handler_ids.destroy({
+            where: {
+                timestamp: { [Models.Sequelize.Op.lt]: timeToBeDeleted },
+                status: { [Models.Sequelize.Op.in]: ['COMPLETED', 'FAILED'] },
+            },
+        });
+
+        const cacheDirectoryPath = this.fileService.getHandlerIdCachePath();
+        if (!fs.existsSync(cacheDirectoryPath)) {
+            return Command.repeat();
+        }
+        const fileList = fs.readdirSync(cacheDirectoryPath);
```

Comment:
Better to use here fs.readdir(cacheDirectoryPath).then();

Reply:
same for lstat and unlink below, if possible

Reply:
start review

Reply:
Updated the whole logic